### PR TITLE
docs(plasma-ui, plasma-web): fix links in docs

### PR DESCRIPTION
**Pull request by kayman233**
:baby_chick: Download canary assets:

[colors--canary.321.4124674935.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.321.4124674935.xml)  
[color_metro_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.321.4124674935.swift)  
[color_metro_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.321.4124674935.kt)  
[color_metro_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.321.4124674935.ts)  
[color_sbermarket_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.321.4124674935.swift)  
[color_sbermarket_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.321.4124674935.kt)  
[color_sbermarket_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.321.4124674935.ts)  
[color_sberprime_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.321.4124674935.swift)  
[color_sberprime_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.321.4124674935.kt)  
[color_sberprime_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.321.4124674935.ts)  
[color_selgros_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.321.4124674935.swift)  
[color_selgros_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.321.4124674935.kt)  
[color_selgros_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.321.4124674935.ts)  
[color_smbusiness_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.321.4124674935.swift)  
[color_smbusiness_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.321.4124674935.kt)  
[color_smbusiness_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.321.4124674935.ts)  
[PlasmaTokensColor--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.321.4124674935.swift)  
[shadow_sbermarket_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.321.4124674935.ts)  
[typo_mage_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.321.4124674935.swift)  
[typo_mage_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.321.4124674935.kt)  
[typo_mage_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.321.4124674935.ts)  
[typo_plasma_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.321.4124674935.swift)  
[typo_plasma_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.321.4124674935.kt)  
[typo_plasma_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.321.4124674935.ts)  
[typo_ruler_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.321.4124674935.swift)  
[typo_ruler_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.321.4124674935.kt)  
[typo_ruler_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.321.4124674935.ts)  
[typo_sage_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.321.4124674935.swift)  
[typo_sage_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.321.4124674935.kt)  
[typo_sage_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.321.4124674935.ts)  
[typo_sbermarket_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.321.4124674935.swift)  
[typo_sbermarket_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.321.4124674935.kt)  
[typo_sbermarket_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.321.4124674935.ts)  
[typo_soulmate_ios-swift--canary.321.4124674935.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.321.4124674935.swift)  
[typo_soulmate_kotlin--canary.321.4124674935.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.321.4124674935.kt)  
[typo_soulmate_react-native--canary.321.4124674935.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.321.4124674935.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-321/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-321/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-321/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-321/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-321/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-321/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-321/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-321/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-321/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-321/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-321/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-321/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-321/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-321/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-321/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-321/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-321/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-321/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-321/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-321/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-321/temple-storybook/

File: .github/workflows/versionate-docs.yml [hunk 1]
```diff
@@ -69,6 +69,7 @@ jobs:
 
       # Без --delete чтобы версии сохранялись
       - name: s3 Upload documentation build
+        if: ${{ steps.versionate_script.outputs.result == 'true' }}
```
[the remark this answers is not in the document]
Comment (Yeti-or) [reply .github/workflows/versionate-docs.yml:72]: Видимо на это: https://github.com/salute-developers/plasma/actions/runs/4124572612/jobs/7123988292#step:10:11

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**